web/pages/create.tsx

```diff
@@ -185,6 +183,22 @@ export function NewContract(props: {
         min < initialValue &&
         initialValue < max))
 
+  const descriptionPlaceholder =
+    outcomeType === 'BINARY'
+      ? `e.g. This question resolves to "YES" if they receive the majority of votes...`
+      : `e.g. I will choose the answer according to...`
+
+  const { editor, upload } = useTextEditor({
+    rows: 3,
+    max: MAX_DESCRIPTION_LENGTH,
+    placeholder: descriptionPlaceholder,
+    disabled: isSubmitting,
+  })
+
+  useWarnUnsavedChanges(
+    !isSubmitting && (Boolean(question) || (editor != null && !editor.isEmpty))
```

Comment:
You could break out a local variable `const editorHasText = editor != null && !editor.isEmpty` or something